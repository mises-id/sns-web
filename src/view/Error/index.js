/*
 * @Author: lmk
 * @Date: 2021-07-15 23:43:29
 * @LastEditTime: 2022-03-24 18:49:07
 * @LastEditors: lmk
 * @Description: my post page
 */
import React from "react";
import "./index.scss";
import { UndoOutline} from "antd-mobile-icons";
const Error = () => {
  const refresh = ()=>{
    window.location.replace('/')
  }
  return (
    <div className="fullpage">
      <div>
        <p className="error-tips">There is a small problem. Please try refreshing</p>
        <div className="refresh-btn" onClick={refresh}>
          <UndoOutline />
          <span>Refresh</span>
        </div>
      </div>
    </div>
  );
};
export default Error;
