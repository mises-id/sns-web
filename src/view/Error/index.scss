.fullpage{
  width: 100vw;
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
}
.error-tips{
  font-size: 16px;
}
.refresh-btn{
  width: 145px;
  height: 50px;
  text-align: center;
  line-height: 50px;
  background: #5D61FF;
  color: white;
  margin: 40px auto;
  font-size: 16px;
  font-weight: bold;
  border-radius: 50px;
  span{
    display: inline-block;
    margin-left: 5px;
  }
}