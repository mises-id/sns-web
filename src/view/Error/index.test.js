/*
 * @Author: lmk
 * @Date: 2021-07-16 00:14:28
 * @LastEditTime: 2021-07-16 00:14:43
 * @LastEditors: lmk
 * @Description: MyLikes test
 */

import React from 'react';
import Error from '.';
import {shallow } from 'enzyme';
describe('Error', () => {
  it('should render correctly', () => {
    const component = shallow(<Error />)
    expect(component).toMatchSnapshot();
  });
});