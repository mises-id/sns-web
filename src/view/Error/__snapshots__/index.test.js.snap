// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<div
  className="fullpage"
>
  <div>
    <p
      className="error-tips"
    >
      There is a small problem. Please try refreshing
    </p>
    <div
      className="refresh-btn"
      onClick={[Function]}
    >
      <UndoOutline />
      <span>
        Refresh
      </span>
    </div>
  </div>
</div>
`;
